#117 changed datetime field (#152)
Element84/stat-api-spec: order/README.md

## Request
| Field Name | Type                                                                       | Description |
| ---------- | -------------------------------------------------------------------------- | ----------- |

> | datetime       | string                                                                     | **REQUIRED.** Single date+time, or a range ('/' separator), formatted to [RFC 3339, section 5.6.](https://tools.ietf.org/html/rfc3339#section-5.6) Use double dots .. for open date ranges. \[Copied from [STAC API spec](https://github.com/radiantearth/stac-api-spec/tree/release/v1.0.0/item-search#query-parameter-table)\] |

Reviewer: ```suggestion
| datetime       | string                                                                     | **REQUIRED.** Single date and time, or a range ('/' separator), formatted to [RFC 3339, section 5.6.](https://tools.ietf.org/html/rfc3339#section-5.6) Use double dots `..` for open date ranges. Only one boundary can use double dots. |
```

Reviewer: What does providing a single datetime mean?

Author: Good question. I imagine it means the same as providing `..` as the second part. But maybe we need to define a new data type here that mandates both parts. *Or*... as previously suggested - make this an array that requires length of 2. This seems like a stronger argument now

Reviewer: For reference: https://docs.ogc.org/is/17-069r3/17-069r3.html#_parameter_datetime @captaincoordinates

Author: @m-mohr PR updated and includes some wording from the OGC Features API spec related to dates (which supports `..` or empty strings for open date ranges).
